### PR TITLE
meson: Restyle meson files to be more readable

### PR DESCRIPTION
**Pull request by Wallbraker**
Meson is just python so format the file with pep8, this goes on top my `jakob-examples-static` branch.

### CODE REVIEW

**Comment by pH5** [comment]
Meson explicitly is not Python, it's a custom "non-Turing complete DSL". Apart from the reasoning I'm not opposed to the change. I see most other projects using 2-space indentation for their meson.build files, though. Is this something we want to imitate?
```
[{meson.build,meson_options.txt}]
indent_style = space
indent_size = 2
```

**Comment by Wallbraker** [comment]
Ah yes so I noticed, I'm not going to dig myself into the indentation size debate so just returned them to the original tab indentation. I mostly just wanted to fix some of the really long lines in the file. Did some more tidying in it as well.

Hopefully we can settle on a indent size in a different PR/Issue.

**Comment by alexandre-janniaux** [comment]
FWIW, dav1d does almost the same [1], so it looks good like this. I only have minor nitpick. 

[1] https://code.videolan.org/videolan/dav1d/blob/master/meson.build#L354

File: meson.build [hunk 1]
```diff
@@ -25,16 +20,31 @@ dep_hidapi = dependency(hidapi)
 dep_threads = dependency('threads')
 
 deps = [
-	meson.get_compiler('c').find_library('m', required : false), #-lm
-	dependency('threads') #pthread
+	meson.get_compiler('c').find_library('m', required: false),  # -lm
```
Comment (alexandre-janniaux) [inline meson.build]: For consistency, you could put this above and have 

```
dep_libm = meson.get_compiler('c').find_library('m', required: false)

deps = [
	dep_threads,
	dep_libm
]
```

Comment (Wallbraker) [reply meson.build]: Done, thanks for the review! :D

**Pull request merged**